content/shell/android/shell_apk/src/org/chromium/content_shell_apk/ContentShellActivity.java

```diff
@@ -163,6 +165,11 @@ protected void onNewIntent(Intent intent) {
         }
     }
 
+    // TODO(cobalt): find another way to set initial url and remove this function.
```

Comment:
Lets change the TODO to an actual ticket from `cobalt` and refer to it in a bug as well